deployment/ansible/development/group_vars/all.travis.yml

```diff
@@ -21,6 +21,9 @@ inasafe:
 
 inasafe_headless_worker:
   command: /docker-entrypoint.sh dev
```

Comment:
make it execute production command: `/docker-entrypoint.sh prod inasafe-headless-worker`